### PR TITLE
use local id in the space range for tag edge index

### PR DESCRIPTION
**Pull request by panda-sheep**
As title

1） Tag, edge, tag index and edge index use space range id.
2） Compatible with the existing old schema creating previously.

Synchronize nebula-storage repo pr https://github.com/vesoft-inc/nebula-storage/pull/501

### CODE REVIEW

**Comment by Shylock-Hg** [comment]
Why need this?

**Comment by panda-sheep** [comment]
> Why need this?
Data synchronization between clusters

**Review by critical27 — APPROVED** [review]
Reviewed before, LGTM.

**Review by bright-starry-sky — APPROVED** [review]

**Comment by panda-sheep** [comment]
Please see the second, compatible with old data

File: src/meta/processors/BaseProcessor-inl.h [hunk 1]
```diff
@@ -170,6 +170,69 @@ ErrorOr<nebula::cpp2::ErrorCode, int32_t> BaseProcessor<RESP>::autoIncrementId()
...
+  auto ret = kvstore_->get(kDefaultSpaceId, kDefaultPartId, localIdkey, &val);
+  if (ret != nebula::cpp2::ErrorCode::SUCCEEDED) {
+    if (ret != nebula::cpp2::ErrorCode::E_KEY_NOT_FOUND) {
+      return ret;
+    }
+
+    // In order to be compatible with the existing old schema, and simple to implement,
+    // when the local_id record does not exist in space, directly use the smallest
+    // id available globally.
+    id = globalId;
```
Comment (Shylock-Hg) [inline src/meta/processors/BaseProcessor-inl.h:215]: I think you could start local id  from 1 when the key don't exists.

Comment (panda-sheep) [reply src/meta/processors/BaseProcessor-inl.h:215]: You can refer to the comments in vesoft-inc/nebula-storage#501

**Review by liuyu85cn — APPROVED** [review]

**Comment by Shylock-Hg** [comment]
So now there are independent incremental id in separate spaces, the schema id will be same for different spaces, and this behavior different with before. For more compatible I think you could reuse the Global ID and just store them in the local id key to distinguish the owned space, and in this case the schema id in different space won't be same

**Review by yixinglu — APPROVED** [review]

**Pull request merged**